### ISSUE TITLE
weird line break behavior

### ISSUE THREAD

**Issue by axl411** [nicklockwood/SwiftFormat#516]
I got this seemly weird line break result, seems like a bug:
```
 ····func blablabla() {
-    ····logger.info("notifyUpdateInfoEvent \(String(describing: connectionInfo)) with response: \(String(describing: response))")
+    ····logger
+        ····.info("notifyUpdateInfoEvent \(String(describing: connectionInfo)) with response: \(String(describing: response))"
+        ····)
```

Version: 
0.42.0

Config:
```
    --cache ignore \
    --disable andOperator,redundantFileprivate,sortedImports,trailingClosures,unusedArguments,redundantRawValues \
    --allman false \
    --binarygrouping none \
    --closingparen balanced \
    --commas always \
    --decimalgrouping 3,4 \
    --elseposition next-line \
    --empty void \
    --exponentcase lowercase \
    --exponentgrouping disabled \
    --fractiongrouping disabled \
    --header ignore \
    --hexgrouping none \
    --hexliteralcase uppercase \
    --ifdef outdent \
    --indent 4 \
    --indentcase false \
    --linebreaks lf \
    --maxwidth 119 \
    --octalgrouping none \
    --operatorfunc spaced \
    --patternlet hoist \
    --ranges nospace \
    --self remove \
    --semicolons inline \
    --stripunusedargs closure-only \
    --trimwhitespace always \
    --wraparguments beforefirst \
    --wrapcollections beforefirst "
```

**Comment by nicklockwood** [comment]
@axl411 this isn't a SwiftFormat config file - maybe you confused it with SwiftLint?

**Comment by axl411** [comment]
> @axl411 this isn't a SwiftFormat config file - maybe you confused it with SwiftLint?
Oops lol, edited.

**Comment by nicklockwood** [comment]
@axl411 this seems to already be fixed in version 0.43.1.
